lib/card.rb

```diff
@@ -0,0 +1,18 @@
+
+# card.rb
+
+class Card
+  attr_reader :value, :suit
+
+  def initialize(value, suit)
+    @value = value
+    @suit = suit
+    
+
```

Comment:
If I understand the specs correctly, the only thing they'll add here is the argument error, correct?